server/src/main/java/org/apache/seata/server/cluster/raft/snapshot/config/ConfigSnapshotFile.java

```diff
@@ -0,0 +1,106 @@
+import org.slf4j.LoggerFactory;
+
+import static org.apache.seata.common.ConfigurationKeys.CONFIG_STORE_TYPE;
+import static org.apache.seata.common.DefaultValues.DEFAULT_DB_TYPE;
+
+public class ConfigSnapshotFile implements Serializable, StoreSnapshotFile {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ConfigSnapshotFile.class);
+
+    private static final long serialVersionUID = 1452307567830545914L;
+
+    private static final Configuration FILE_CONFIG = ConfigurationFactory.CURRENT_FILE_INSTANCE;
+
+    private static ConfigStoreManager configStoreManager;
+    String group;
+
+    String fileName = "config";
+
+    public ConfigSnapshotFile(String group) {
+        this.group = group;
+        String dbType = FILE_CONFIG.getConfig(CONFIG_STORE_TYPE, DEFAULT_DB_TYPE);
+        configStoreManager = EnhancedServiceLoader.load(ConfigStoreManagerProvider.class, Objects.requireNonNull(dbType), false).provide();
+    }
+
+    @Override
+    public Status save(SnapshotWriter writer) {
+        Map<String, Map<String, Object>> configMap = configStoreManager.getConfigMap();
+        RaftSnapshot raftSnapshot = new RaftSnapshot();
+        raftSnapshot.setBody(configMap);
+        raftSnapshot.setType(RaftSnapshot.SnapshotType.config);
+        LOGGER.info("groupId: {}, config size: {}", group, configMap.size());
+        String path = new StringBuilder(writer.getPath()).append(File.separator).append(fileName).toString();
+        try {
+            if (save(raftSnapshot, path)) {
+                if (writer.addFile(fileName)) {
+                    return Status.OK();
+                } else {
+                    return new Status(RaftError.EIO, "Fail to add file to writer");
+                }
+            }
+        } catch (IOException e) {
+            LOGGER.error("Fail to save groupId: {} snapshot {}", group, path, e);
+        }
+        return new Status(RaftError.EIO, "Fail to save groupId: " + group + " snapshot %s", path);
+    }
+
+    @Override
+    public boolean load(SnapshotReader reader) {
+        if (reader.getFileMeta(fileName) == null) {
+            LOGGER.error("Fail to find data file in {}", reader.getPath());
+            return false;
+        }
+        String path = new StringBuilder(reader.getPath()).append(File.separator).append(fileName).toString();
+        try {
+            LOGGER.info("on snapshot load start index: {}", reader.load().getLastIncludedIndex());
+            Map<String, Map<String, Object>> configMap = (Map<String, Map<String, Object>>)load(path);
+            ConfigStoreManager configStoreManager = RocksDBConfigStoreManager.getInstance();
+            configStoreManager.clearData();
+            configStoreManager.putConfigMap(configMap);
+            if (LOGGER.isInfoEnabled()) {
+                LOGGER.info("on snapshot load end index: {}", reader.load().getLastIncludedIndex());
+            }
+            return true;
+        } catch (final Exception e) {
```

Comment:
Is 'final' here necessary?

Reply:
just consistent with other snapshot class code, or uniformly remove the 'final' in all snapshot class